### PR TITLE
readded loki script and altered read.me to reflect

### PR DESCRIPTION
**Pull request by lauren-mieczkowski-dish**
added loki sh file and altered the read.me to reflect the ability to use this.

### CODE REVIEW

File: napp/monitoring/deploy-loki.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#Set the context for the right cluster and namespace
+aws eks update-kubeconfig --region $cluster_region --name $cluster_name
+kubectl config set-context --current --namespace=$cluster_namespace
+
+
+#Installing loki
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm upgrade --install loki-stack grafana/loki-stack \
+    --set promtail.enabled=false,grafana.enabled=true
```
Comment (PierceLovesee) [inline napp/monitoring/deploy-loki.sh:24]: Please alter to add Fluentbit

**Review by PierceLovesee — CHANGES_REQUESTED** [review]
please see changes for deploy-loki.sh

File: napp/monitoring/deploy-loki.sh (hunk 1 above)
Comment (lauren-mieczkowski-dish) [reply napp/monitoring/deploy-loki.sh]: added fluent-bit.enabled=true

**Review by PierceLovesee — APPROVED** [review]
Thanks for adding that .sh back in!

**Pull request merged**